lib/api/gitlab.js

```diff
@@ -275,9 +284,14 @@ async function mergePr(pr) {
 // Generic File operations
 
 async function getFile(filePath, branchName = config.defaultBranch) {
-  const res = await glGot(
-    `projects/${config.repoName}/repository/files?file_path=${filePath}&ref=${branchName}`
-  );
+  // Gitlab API v3 support
+  let url;
```

Comment:
You could also make these ternary statements, which would avoid some of the syntax overhead, and allow you to set `url` to `const`.